src/collatzConjecture.js

```diff
@@ -0,0 +1,12 @@
+export default function collatzConjecture(n){
+  let numArray = [n]
+  function recursiveCall(arr){
+    if (arr[arr.length-1] === 1){
```

Comment:
Since you are looking at `arr[arr.length -1]` in every branch of your `if` block, you should capture that in a variable in the beginning of the function, and use it everywhere else.